Source/Request.swift

```diff
@@ -506,6 +515,14 @@ open class Request {
         protectedMutableState.write { $0.redirectHandler = handler }
         return self
     }
+
+    // MARK: - Cached Responses
+
+    @discardableResult
+    open func cacheResponse(with handler: CachedResponseHandler) -> Self {
```

Comment:
I prefer `using` as the modifier for methods like this. e.g. `.cacheResponse(using: myHandler)`.

Reply:
Updated in efa2a1db.